Source/JavaScriptCore/runtime/ProgramExecutable.cpp

```diff
@@ -101,7 +101,7 @@ JSObject* ProgramExecutable::initializeGlobalProperties(VM& vm, JSGlobalObject*
         // Check for intersection of "var" and "let"/"const"/"class"
         for (auto& entry : lexicalDeclarations) {
             if (variableDeclarations.contains(entry.key))
-                return createSyntaxError(globalObject, makeString("Can't create duplicate variable: '", String(entry.key.get()), "'"));
+                return createSyntaxError(globalObject, makeString("Can't create duplicate variable: '"_s, String(entry.key.get()), '\''));
```

Comment:
Should be able to pass `entry.key` without `String` to avoid reference count churn.